pkg/k8shandler/cluster.go

```diff
@@ -59,14 +59,18 @@ func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateElasticsearchClu
 
 		if _, ok := containsNodeTypeInterface(upgradeInProgressNode, scheduledUpgradeNodes); ok {
 			logrus.Debugf("Continuing update for %v", upgradeInProgressNode.name())
-			upgradeInProgressNode.update(nodeStatus)
+			if err := upgradeInProgressNode.update(nodeStatus); err != nil {
+				return err
+			}
 		} else {
 			logrus.Debugf("Continuing restart for %v", upgradeInProgressNode.name())
 			upgradeInProgressNode.rollingRestart(nodeStatus)
 		}
 
 		addNodeState(upgradeInProgressNode, nodeStatus)
-		elasticsearchRequest.setNodeStatus(upgradeInProgressNode, nodeStatus, clusterStatus)
+		if err := elasticsearchRequest.setNodeStatus(upgradeInProgressNode, nodeStatus, clusterStatus); err != nil {
+			return err
```

Comment:
same here

Reply:
Fixed by printing the error instead of returning it.